rlkit/torch/model_based/dreamer/experiments/experiment_utils.py

```diff
@@ -49,7 +49,7 @@ def preprocess_variant_llraps(variant):
     ]
 
     variant["replay_buffer_kwargs"]["max_replay_buffer_size"] = int(
-        3e6
+        3e4
```

Comment:
why was this number changed?

Reply:
It was changed because the current number would cause it to allocate ~60 GB tensor leading to an out of memory error.

Reply:
okay then wrap it in an if block instead of changing the default value (see the point above regarding if block for use wrist cam)

Reply:
Done, I have account for that as well.